Revised ManualResetEvent remarks

pkulikov opened the pull request:
Accompanies dotnet/docs#10233
/cc: @rpetrusha

rpetrusha commented on xml/System.Threading/ManualResetEventSlim.xml:
```diff
@@ -48,15 +48,15 @@
    
```
Unrelated to your PR: The type description, "Provides a slimmed down version of ManualResetEvent", isn't helpful. Could you expand it by adding "to represent a thread synchronization event that, when signaled, must be reset manually."?

pkulikov commented:
@rpetrusha thank you for review and suggestions. I've addressed all the feedback (also replaced the "slimmed down" words with "lightweight", as it looks to be more common in the docs so far).
